src/components/Alert/index.tsx

```diff
@@ -96,7 +73,7 @@ const StyledContainer = styled(Box, {
   align-items: center;
   border-radius: ${({ theme }) => theme.radii.default};
   padding: 12px;
-  ${variantStyles}
+  ${({ theme, type, variant }) => alertStyles(theme, type)[variant]}
```

Comment:
Might be good to avoid the double function call and let the object access in alertStyles ?

Reply:
fixed :+1: